web/styles/zulip.css

```diff
@@ -2628,6 +2628,10 @@ select.invite-as {
     text-transform: uppercase;
 }
 
+.date_row span {
```

Comment:
Hey, I noticed something interesting! It seems we also have styles for the `span` inside the `.date_row` class in both `message_row.css` and `message_edit_history.css`. Do you think it would be a good idea to bring them together and apply them in a specific place?

Reply:
Excellent point. I'll revise the PR here with a commit that does some reorg and consolidation prior to the one that'll have the padding fix.

Reply:
All right. I think would take more finesse than this PR requires, but I've added the padding now to the more specific location in `message_row.css` (the display of `.date_row` in message history is a bit different, but both it and the message-row version draw on some of the shared styles in `zulip.css`).